src/common/virtual_file_util.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2019 yuzu Emulator Project
+// Licensed under GPLv2 or any later version
+// Refer to the license.txt file included.
+
+#include "virtual_file_util.h"
```

Comment:
Project includes should always be relative to the source root.

```suggestion
#include "common/virtual_file_util.h"
```